genesyscloud/task_management_workitem/resource_genesyscloud_task_management_workitem.go

```diff
@@ -0,0 +1,177 @@
+package task_management_workitem
+
+import (
+	"context"
+	"fmt"
+	"log"
+	resourceExporter "terraform-provider-genesyscloud/genesyscloud/resource_exporter"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/mypurecloud/platform-client-sdk-go/v119/platformclientv2"
+
+	"terraform-provider-genesyscloud/genesyscloud/consistency_checker"
+
+	gcloud "terraform-provider-genesyscloud/genesyscloud"
+
+	"terraform-provider-genesyscloud/genesyscloud/util/resourcedata"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/retry"
+)
+
+/*
+The resource_genesyscloud_task_management_workitem.go contains all of the methods that perform the core logic for a resource.
+*/
+
+// getAllAuthTaskManagementWorkitem retrieves all of the task management workitem via Terraform in the Genesys Cloud and is used for the exporter
+func getAllAuthTaskManagementWorkitems(ctx context.Context, clientConfig *platformclientv2.Configuration) (resourceExporter.ResourceIDMetaMap, diag.Diagnostics) {
+	proxy := newTaskManagementWorkitemProxy(clientConfig)
+	resources := make(resourceExporter.ResourceIDMetaMap)
+
+	workitems, err := proxy.getAllTaskManagementWorkitem(ctx)
+	if err != nil {
+		return nil, diag.Errorf("Failed to get task management workitem: %v", err)
+	}
+
+	for _, workitem := range *workitems {
+		resources[*workitem.Id] = &resourceExporter.ResourceMeta{Name: *workitem.Id}
```

Comment:
I think it should be `Name: *workItem.Name` otherwise when exported the name of the resource will be a guid

Reply:
Fixed